package.json

```diff
@@ -68,15 +70,15 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "^1.1.1",
-    "@openedx/frontend-build": "^13.0.28",
+    "@openedx/frontend-build": "npm:@raccoongang/frontend-build@^13.1.4-tokens.2",
     "@testing-library/react": "12.1.5",
     "axios": "0.21.2",
     "axios-mock-adapter": "^1.17.0",
     "es-check": "^2.3.0",
     "fetch-mock": "^6.5.2",
     "husky": "2.7.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.6.3",
+    "jest": "^29.7.0",
```

Comment:
[question]: Do I understand correctly that we raised the version of jest due to conflicts with third-party packages? (e.g. @edx/react-unit-test-utils)

Reply:
Yes, you are right.